### DISCUSSION TITLE
Firmware update problems

### DISCUSSION THREAD

**Discussion by BlueScreenTT** [Koenkk/zigbee2mqtt#10721] [General]
What is happenig here?

![Screenshot_20220111-172941_Chrome](https://user-images.githubusercontent.com/89699439/148982719-880c3d51-0f98-4096-ba10-ffe09a897b4e.jpg)

Been like this for 1 day

Before that it was like this for 1 day
![image](https://user-images.githubusercontent.com/89699439/148982933-bdc91ba3-0e1a-4a9c-87bd-352083410b9f.png)

**Comment by groenmarsmannetje** [comment]
Only run one firmware update at a time. Otherwise it will flood your zigbee network and makes it unresponsive. 

See [documentation](https://www.zigbee2mqtt.io/guide/usage/ota_updates.html#update-to-latest-firmware)

**Comment by BlueScreenTT** [comment]
Ok   thank you 😀

Den tors 13 jan. 2022 15:35groenmarsmannetje ***@***.***>
skrev:

**Comment by BlueScreenTT** [comment]
yea

so it still dont work

it has been like this for 12 hours now

![Z2M FW UPDATE](https://user-images.githubusercontent.com/89699439/149498976-fe21fc76-3e6a-49af-abe5-1955deda752b.jpg)
